crates/miden-objects/src/transaction/partial_blockchain.rs

```diff
@@ -426,6 +429,8 @@ mod tests {
         let fee_parameters =
             FeeParameters::new(ACCOUNT_ID_PUBLIC_FUNGIBLE_FAUCET.try_into().unwrap(), 500)
                 .expect("native asset ID should be a fungible faucet ID");
+        let mut rng = ChaCha20Rng::from_seed([0u8; 32]);
+        let validator_key = SecretKey::with_rng(&mut rng).public_key();
```

Comment:
can we call `SecretKey::new()` instead?

Reply:
That requires std and would also randomize the seed which we want to  avoid for the tests here